src/services/axios.ts

```diff
@@ -0,0 +1,45 @@
+import axios, { AxiosResponse } from 'axios';
+
+axios.defaults.baseURL = process.env.REACT_APP_API_BASE_URL;
+axios.defaults.headers.post['Content-Type'] = 'application/json';
```

Comment:
Using default url from env properties. Also, _pretty_ sure the best default content type will be JSON... but not totally sure.